docs/comparison.adoc

```diff
@@ -113,9 +113,9 @@ In bold : Implementations that differ between flavors or between versions
 |===
 |Specifications|Implementations alternatives +
 //(see icons for compatibilities with TomEE 9.x)
-|Jakarta Persistence (JPA)|https://hibernate.org/orm/[Hibernate ORM^] {y} 6.1.x +
+|Jakarta Persistence (JPA)|https://hibernate.org/orm/[Hibernate ORM^] {y} 6.6.x
 |Jakarta MVC|
-https://eclipse-ee4j.github.io/krazo/[Eclipse Krazo^] {y} 2.0.x +
+https://eclipse-ee4j.github.io/krazo/[Eclipse Krazo^] {y} 3.0.x +
```

Comment:
Krazo 3.0.0 dropped CXF support (https://github.com/eclipse-ee4j/krazo/commit/96a839e692a863e591d82d2d55d6c438902645d4), we should probably not mention it anymore